admin/src/components/field.tsx

```diff
@@ -119,6 +120,7 @@ interface FieldExtraProps extends Props {
 
 function FieldExtra({ type, name }: FieldExtraProps) {
   const { control, register, watch } = useFormContext()
+  const [chipInputState, setChipInputState] = React.useState(true)
```

Comment:
```suggestion
  const [showEnumInput, setShowEnumInput] = React.useState(true)
```

Reads better

Reply:
The enum input should be hidden when no value is available by default.

When there are values, it should show by default.

Note that, there can be values already when in edit mode (ie, editing a collection where a field already has enums)